### PR TITLE
Re-factored test cases and class names

### PR DESCRIPTION
**Pull request by kangzongxian**
This is to make it sound more in line with our
application.

This makes our code base more readable.

### CODE REVIEW

File: src/test/java/seedu/guest/testutil/GuestBookBuilder.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
+package seedu.guest.testutil;
+
+import seedu.guest.model.GuestBook;
+import seedu.guest.model.guest.Guest;
+
+/**
+ * A utility class to help with building Guestbook objects.
+ * Example usage: <br>
+ *     {@code GuestBook ab = new GuestBookBuilder().withPerson("John", "Doe").build();}
```
Comment (jehousoh) [inline src/test/java/seedu/guest/testutil/GuestBookBuilder.java]: Should this be withGuest?

File: src/test/java/seedu/guest/testutil/GuestBookBuilder.java [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    }
+
+    public GuestBookBuilder(GuestBook guestBook) {
+        this.guestBook = guestBook;
+    }
+
+    /**
+     * Adds a new {@code Guest} to the {@code GuestBook} that we are building.
+     */
+    public GuestBookBuilder withPerson(Guest guest) {
```
Comment (jehousoh) [inline src/test/java/seedu/guest/testutil/GuestBookBuilder.java]: Should this be withGuest?

**Review by jehousoh — CHANGES_REQUESTED** [review]
Looks good with some minor changes!

File: src/test/java/seedu/guest/testutil/GuestBookBuilder.java (hunk 1 above)
Comment (kangzongxian) [reply src/test/java/seedu/guest/testutil/GuestBookBuilder.java]: Agreed!

File: src/test/java/seedu/guest/testutil/GuestBookBuilder.java (hunk 2 above)
Comment (kangzongxian) [reply src/test/java/seedu/guest/testutil/GuestBookBuilder.java]: Agreed!

**Review by jehousoh — APPROVED** [review]
lgtm!

File: src/test/java/seedu/guest/storage/StorageManagerTest.java [hunk 3]
```diff
@@ -54,7 +54,7 @@ public void addressBookReadSave() throws Exception {
          * {@link JsonAddressBookStorage} class.
```
Comment (Lan-Jingbo) [inline src/test/java/seedu/guest/storage/StorageManagerTest.java]:
```suggestion
         * {@link JsonGuestBookStorage} class.
```

File: src/test/java/seedu/guest/storage/StorageManagerTest.java [hunk 4]
```diff
@@ -54,7 +54,7 @@ public void addressBookReadSave() throws Exception {
...
          * More extensive testing of UserPref saving/reading is done in {@link JsonAddressBookStorageTest} class.
```
Comment (Lan-Jingbo) [inline src/test/java/seedu/guest/storage/StorageManagerTest.java]:
```suggestion
         * More extensive testing of UserPref saving/reading is done in {@link JsonGuestBookStorageTest} class.
```

**Review by Lan-Jingbo — CHANGES_REQUESTED** [review]

File: src/test/java/seedu/guest/storage/StorageManagerTest.java (hunk 3 above)
Comment (kangzongxian) [reply src/test/java/seedu/guest/storage/StorageManagerTest.java]: Agreed!

File: src/test/java/seedu/guest/storage/StorageManagerTest.java (hunk 4 above)
Comment (kangzongxian) [reply src/test/java/seedu/guest/storage/StorageManagerTest.java]: Agreed!

**Review by Lan-Jingbo — APPROVED** [review]
LGTM

**Pull request merged**